pytorch_lightning/core/lightning.py

```diff
@@ -171,7 +171,7 @@ def datamodule(self) -> Any:
         warning_cache.deprecation(
             "The `LightningModule.datamodule` property is deprecated in v1.3 and will be removed in v1.5."
             " Access the datamodule through using `self.trainer.datamodule` instead.",
-            stacklevel=5,
+            stacklevel=6,
```

Comment:
mind write a comment on what the 6 means or refer to some docs?

Reply:
It's from python

> The third optional keyword argument is stacklevel, which defaults to 1. If greater than 1, the corresponding number of stack frames are skipped when computing the line number and function name set in the LogRecord created for the logging event. This can be used in logging helpers so that the function name, filename and line number recorded are not the information for the helper function/method, but rather its caller.

https://docs.python.org/3/library/logging.html#logging.Logger.debug